app/models/cloud_service.rb

```diff
@@ -0,0 +1,37 @@
+class CloudService < ApplicationRecord
+  belongs_to :ext_management_system, :foreign_key => :ems_id, :class_name => "ManageIQ::Providers::CloudManager"
+  belongs_to :host
+  belongs_to :system_service
+  belongs_to :availability_zone
+
+  def fog_service
+    connection_options = {:service => source}
+    ext_management_system.with_provider_connection(connection_options) do |service|
+      service.services.find { |s| s.id.to_s == ems_ref }
```

Comment:
Hrm, I was hoping the [Fog OpenStack API](https://github.com/fog/fog-openstack/tree/8b1345380f1186cda4fbed83cc9bdc37042b8c6d/lib/fog/openstack/requests/compute) would have a call to get just one service by ID.  Seems not.  :disappointed: 

Might be something to push back into Fog (and possibly needs to be added to nova as well).